core/dbt/parser/generic_test_builders.py

```diff
@@ -25,9 +25,12 @@
 from dbt.parser.search import FileBlock
 
 
-def get_nice_generic_test_name(
+def synthesize_generic_test_names(
     test_type: str, test_name: str, args: Dict[str, Any]
 ) -> Tuple[str, str]:
+    # Using the type, name, and arguments to this generic test, synthesize a (hopefully) unique name
```

Comment:
[nit] If we're going to use the word "hopefully" in this comment let's lay out exactly when this would not be unique.

Reply:
thanks for adding that!